neuro_evolution_ctrnn/tools/episode_runner.py

```diff
@@ -1,18 +1,20 @@
-from gym import wrappers
+import gym
 import time
 import logging
 from bz2 import BZ2Compressor
 import numpy as np
 
 from tools.helper import set_random_seeds, output_to_action
 from tools.configurations import EpisodeRunnerCfg, IBrainCfg
-from tools.dask_handler import get_current_worker
 from tools.env_handler import EnvHandler
 from brains.continuous_time_rnn import ContinuousTimeRNN
 from brains.CNN_CTRNN import CnnCtrnn
 
 
 class EpisodeRunner:
+
+    _env: gym.Env = None
```

Comment:
wäre es sinnvoll das als property zu machen, damit der aufrufer sich nicht um das singleton-handlign kümmern braucht? 

also z.b. sowas

```python3
    _env: Optional[gym.Env] = None
    @property
    def env(self) -> gym.Env:
        if EpisodeRunner._env is None:
            EpisodeRunner._env = self.env_handler.make_env(self.env_id)
        else:
            assert self.env_id.split(":")[-1] == EpisodeRunner._env # das split behandelt den sonderfall von procgen
            return EpisodeRunner._env
```

hier ist zusätzlich noch ein safeguard, der sicherstellt, dass in der zwischenzeit kein seiteneffekt dafür gesorgt hat, dass die env nicht mehr die richtige ist.


`Optional` kommt aus `from typing import Optional`